benchmark/multi_exp_pip/benchmark.m.cc

```diff
@@ -228,19 +232,26 @@ int main(int argc, char* argv[]) {
 
   if (curve_str == "curve25519") {
     std::println("running {} benchmark...", curve_str);
-    auto accessor = make_partition_table_accessor<c21t::element_p3>(n, curve25519_generator);
+#if 0
+    auto accessor = make_partition_table_accessor<c21t::element_p3, c21t::element_p3>(n, curve25519_generator);
```

Comment:
Do you see adding an argument to the benchmark to choose between compact and non-compact representations in the future? If not, maybe we should just remove this block of code?

Reply:
Sorry. Forgot to remove that. Fixed.